app/models/manageiq/providers/microsoft/infra_manager.rb

```diff
@@ -24,8 +24,7 @@ def self.raw_connect(connect_params)
     require 'winrm'
 
     connect_params[:operation_timeout] = 1800
-    winrm = WinRM::Connection.new(connect_params)
-    winrm
+    WinRM::Connection.new(connect_params)
   end
```

Comment:
@Fryguy I don't call those methods directly. We are using with_provider_connection for the actual calls. That just hooks into the connect/raw_connect methods, right?